### PR TITLE
Add the ability to lose focus

### LINKED ISSUE #1308: Add an unfocus API
/cc @Hixie

### PR DESCRIPTION
**Pull request by abarth**
Fixes #1308

### CODE REVIEW

**Comment by abarth** [comment]
@collinjackson

File: packages/flutter/lib/src/widgets/focus.dart [hunk 1]
```diff
@@ -144,6 +144,12 @@ class Focus extends StatefulComponent {
     }
   }
 
+  static void blur(BuildContext context) {
```
Comment (collinjackson) [inline packages/flutter/lib/src/widgets/focus.dart]: Hixie wanted to call this clear() I believe

Comment (abarth) [inline packages/flutter/lib/src/widgets/focus.dart]: Will do.

**Comment by collinjackson** [comment]
lgtm

**Comment by abarth** [comment]
Thanks!

**Pull request merged**